### DISCUSSION TITLE
how to make gq/gw  indent markdown file correctly?

### DISCUSSION THREAD

**Discussion by ssh352** [LazyVim/LazyVim#2916] [Q&A] [answered]
Hello, could you advise why gq or gw command won't indent the following markdown file correctly? I wish the word `indented` is indented just as I used https://github.com/nvim-lua/kickstart.nvim. 

actually the indentation works with gq without any configuration files `nvim -u NONE`

I currently use https://github.com/LazyVim/starter without modification.
Thanks!

```
abc
- To be called when a new leader event is delivered. This method needs to be
indented when using the parseIngressEndpoints(final String endpoints)
- next item in list
```

**Comment by andreacfromtheapp** [comment]
I use `gwap` for paragraph indent fixes. I would love `markdownlint` to do the job for me but I haven't managed to make it work.

**Reply by ssh352** [reply]
`gwap` doesn't work for me. the word `indented` should be indented in the above example, but `gwap` doesn't do it.

**Comment by dpetka2001** [comment, marked as answer]
It's the `indent` setting of `nvim-treesitter`. `kickstart.nvim` also behaves the same as LazyVim (just tested it). If you disable `indent` from `nvim-treesitter`, it will behave the same as when you do `nvim -u NONE` (you can also disable `indent` for specific languages only).
